assets/js/modules/optimize/setup.js

```diff
@@ -151,7 +151,7 @@ class OptimizeSetup extends Component {
 				finishSetup();
 			}
 
-			getModulesData().optimize.settings.optimizeID = optimizeID;
+			getModulesData().optimize.settings = optimizeAccount;
```

Comment:
This is very temporary and all but can we set the settings with the response from the server? 😄 

E.g. https://github.com/google/site-kit-wp/blob/8e56c5810cc3dc2a1ac40b8e48e27316e6e4a822/assets/js/modules/tagmanager/setup.js#L363-L369

We should be able to get the `savedSettings` from the value provided to `.then()`

Reply:
Hah, sure thing, my bad 😅